libraries/testing/tester.cpp

```diff
@@ -20,6 +20,32 @@ eosio::chain::asset core_from_string(const std::string& s) {
 }
 
 namespace eosio { namespace testing {
+
+   std::ostream& operator<<(std::ostream& os, setup_policy p) {
```

Comment:
Does os need to be initialized?

Reply:
I don't understand the question.

Reply:
What I meant what happens if `os` has already contained some old value? Should it be set to empty before the first `<<`?